vnext/src/Libraries/Components/Touchable/TouchableOpacity.windows.js

```diff
@@ -163,21 +163,31 @@ class TouchableOpacity extends React.Component<Props, State> {
         }
       },
       onLongPress: this.props.onLongPress,
-      onPress: this.props.onPress,
+      onPress: (event) => {
+        if (this.props.onPress != null && event.nativeEvent.isLeftButton) {
+          this.props.onPress(event);
+        }
```

Comment:
Touchable, Pressable, etc internally use a library called `Pressability` (forked at `vnext/src/Libraries/Pressability/Pressability.windows.js`). Instead of adding this check to each of the users of `Pressability`, would fixing there work for all of these cases?

Reply:
I thought the only components we didn't want to register right-click were Touchable, Pressable, and Button. Both Text and TextInput use `Pressability`, if we also don't want those to register right-clicks, we could add the check to `Pressability`

Reply:
I think that should be okay and would match the current `TextInput` behavior as well, since it doesn't respond to non-left-click today.